config/initializers/job_configurations.rb

```diff
@@ -127,6 +127,11 @@
         cron: cron_1h,
         args: -> { [Time.zone.now - 1.hour] },
       },
+      # Weekly IRS report returning system demand
+      irs_weekly_summary_report: {
+        class: 'IrsWeeklySummaryReport',
+        cron: cron_24h,
```

Comment:
the report's title is "weekly" report but this job runs every day... do we want a version that only runs once a week? if so, we'll need a new cron string for it

Reply:
oh okay I was confused because I saw gpo_monthly_letters had `cron_24h ` yes I want it to run once a week. will update